packages/microcosm-www-next/src/pages/index.js

```diff
@@ -39,14 +40,20 @@ export default class IndexPage extends React.Component {
 
   onIntersection = observed => {
     let entry = observed[0]
-    let section = entry.target.dataset.section
+    let section = parseInt(entry.target.dataset.section)
     let notAlreadyVisible = section !== this.state.currentSection
 
-    if (entry.isIntersecting & notAlreadyVisible) {
+    if (entry.isIntersecting && notAlreadyVisible) {
+      this.changeBgColor(this.state.currentSection, section)
       this.setState({ currentSection: section })
     }
   }
 
+  changeBgColor(oldSection, newSection) {
+    this.body.classList.remove(`bg-color-${oldSection}`)
+    this.body.classList.add(`bg-color-${newSection}`)
+  }
```

Comment:
Any concerns with the background color staying the section color when the user transitions to another page? Are there any other pages?

Reply:
there are no other pages so should be okay! I'll keep an eye on that